src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkUnrecognizedSectionInspection.kt

```diff
@@ -8,11 +8,17 @@ import com.intellij.codeInspection.ui.ListEditForm
 import com.jetbrains.snakecharm.SnakemakeBundle
 import com.jetbrains.snakecharm.codeInsight.SnakemakeAPI.MODULE_SECTIONS_KEYWORDS
 import com.jetbrains.snakecharm.codeInsight.SnakemakeAPI.RULE_OR_CHECKPOINT_ARGS_SECTION_KEYWORDS
+import com.jetbrains.snakecharm.codeInsight.SnakemakeAPI.RULE_OR_CHECKPOINT_SECTION_KEYWORDS
 import com.jetbrains.snakecharm.codeInsight.SnakemakeAPI.SUBWORKFLOW_SECTIONS_KEYWORDS
 import com.jetbrains.snakecharm.codeInsight.SnakemakeAPI.USE_SECTIONS_KEYWORDS
 import com.jetbrains.snakecharm.lang.psi.*
 import com.jetbrains.snakecharm.lang.psi.elementTypes.SmkElementTypes
 import com.jetbrains.snakecharm.inspections.quickfix.AddIgnoredElementQuickFix
+import com.jetbrains.snakecharm.lang.SnakemakeNames.CHECKPOINT_KEYWORD
+import com.jetbrains.snakecharm.lang.SnakemakeNames.MODULE_KEYWORD
+import com.jetbrains.snakecharm.lang.SnakemakeNames.RULE_KEYWORD
+import com.jetbrains.snakecharm.lang.SnakemakeNames.SUBWORKFLOW_KEYWORD
+import com.jetbrains.snakecharm.lang.SnakemakeNames.USE_KEYWORD
 import javax.swing.JComponent
 
 class SmkUnrecognizedSectionInspection : SnakemakeInspection() {
```

Comment:
According to the inspection description it highligts section keywords not supported by the plugin. I think #408 should be an independent inspection. More over in case of #408 we are pretty sure that we will get runtime error, thus it should be ERROR, not weak warning.